Sources/KSCrashInstallations/include/KSCrashInstallation.h

```diff
@@ -38,15 +40,16 @@
  *
  * This is an abstract class.
  */
+NS_SWIFT_NAME(Installation)
```

Comment:
Maybe it's better to be a bit more specific?
```suggestion
NS_SWIFT_NAME(CrashInstallation)
```